exodus_lambda/functions/base.py

```diff
@@ -18,22 +18,12 @@ def conf(self):
             if isinstance(self._conf_file, dict):
                 self._conf = self._conf_file
             else:
-                for conf_file in [
-                    self._conf_file,
-                    os.path.join(
-                        os.path.dirname(
-                            os.path.dirname(os.path.dirname(__file__))
-                        ),
-                        "configuration",
-                        "lambda_config.json",
-                    ),
-                ]:
-                    if os.path.exists(conf_file):
-                        with open(
-                            conf_file, "r", encoding="UTF-8"
-                        ) as json_file:
-                            self._conf = json.load(json_file)
-                        break
+                # to be polished: what if the self._conf_file not exist
```

Comment:
How about just don't do the os.path.exists check?  If the config file doesn't exist, just let the open() call crash - what else can you do?  If you added custom error handling, you'd just be obscuring the error.